SU-HSDO/suhumsci
# HSD8-1091: Added year conversion to dateTime widget for years before 1969.

# READY FOR REVIEW

## Summary
- Fixed absolute years not working in DateTimeYearOnly widget
- This had to do with the fact they were using 1949 specifically -- the DateTime object uses timestamp conversion so any year before `december 31 1969` won't work, and needs to be formatted as `1949-06-01` instead of just `1949`. Years after 1969 work fine.

## Urgency
Medium

## Steps to Test
1. Pull/setup locally
2. Open `/admin/structure/types/manage/hs_person/form-display` and set the Cohort field Min setting before 1970
3. Open `/node/add/hs_person`, go to Student Info tab, and check Cohort field min is set correctly

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
